apps/api/app/services/tracked_todo_service.py

```
@@ -0,0 +1,400 @@
+                    )
+                    if canvas:
+                        # Extract Key Details section
+                        match = re.search(
+                            r"## Key Details\n(.*?)(?=\n## |\Z)",
+                            canvas,
+                            re.DOTALL,
+                        )
+                        if match:
+                            key_details = match.group(1).strip()
+                except Exception:
+                    pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)